src/controllers/image-request.ts

```diff
@@ -111,17 +90,47 @@ export const handle = async (
     };
   }
 
+  const getFromExternal = TASK_PARAMETERS.GET_FROM_EXTERNAL === 'true';
+  const baseExternalUrl = TASK_PARAMETERS.BASE_EXTERNAL_URL;
+
+  let imagePath = request.uri;
```

Comment:
Being a bit picky now but we should allow the image path to flow through to the repository layer and instead use the https://axios-http.com/docs/req_config `baseUrl` config and set it to `TASK_PARAMETERS.BASE_EXTERNAL_URL`.

That way we can remove the isExternalUrl and instead rely solely on the  `TASK_PARAMETERS.GET_FROM_EXTERNAL`

Reply:
Means we don't worry about this value until we actually need it (the repository layer)